cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -911,17 +911,17 @@
   (if-let $false(has_b))
   (gen_cltz $false x ty))
 
-(rule
+(rule 2
   (lower_ctz $I64 x)
   (if-let $true(has_b))
```

Comment:
pre-existing but can we fix the formatting here while we're here (space after `$true`) and twice below?